### ISSUE TITLE
Details of the loss function used during training

### ISSUE THREAD

**Issue by woshixyr** [LianpoWang/CAMEW-UNet#17]
Which loss function was mainly used during the model training? Were multiple loss functions combined to optimize the depth estimation effect?
